src/content/symbol-store-db.js

```diff
@@ -65,26 +65,29 @@ export class SymbolStoreDB {
           reject(openReq.error);
         }
       };
-      openReq.onupgradeneeded = upgradeEvent => {
+      openReq.onupgradeneeded = ({ oldVersion }) => {
         const db = openReq.result;
         db.onerror = reject;
-        if (upgradeEvent.oldVersion === 1) {
+
+        if (oldVersion === 1) {
           db.deleteObjectStore('symbol-tables');
         }
         const store = db.createObjectStore('symbol-tables', {
           keyPath: ['debugName', 'breakpadId'],
         });
         store.createIndex('lastUsedDate', 'lastUsedDate');
       };
+
       openReq.onblocked = () => {
         reject(new Error(
           'The symbol store database could not be upgraded because it is ' +
           'open in another tab. Please close all your other perf-html.io ' +
           'tabs and refresh.'));
       };
+
       openReq.onsuccess = () => {
         const db = openReq.result;
-        db.onupgradeneeded = () => {
+        db.onversionchange = () => {
```

Comment:
I think this is what you intended. `upgradeneeded` is for the open request, while `versionchange` is for the DB.